src/App.js

```diff
@@ -1,23 +1,38 @@
-import logo from './logo.svg';
+import {Route, Routes} from 'react-router-dom'
+import { useEffect, useState } from "react";
+import axios from 'axios'
+
+
+
 import './App.css';
+import Homepage from "./components/Homepage";
+import Navbar from './components/Navbar.js';
+import AllBeers from './components/AllBeers'
+import Beer from './components/Beer'
+import NewBeer from './components/NewBeer' 
 
 function App() {
+
+  let [beers, setBeers] = useState([])
+
+    useEffect( () => {
+        axios.get('https://ih-beers-api2.herokuapp.com/beers')
+            .then( response => {
+                 setBeers(response.data)
+
+                })
+            .catch( err => console.log(err))
+    }, [])
```

Comment:
Currently, when I create new beer, I need to refresh page in order to see newly added item. To avoid that, every time we create new beer, we want to get new updated beersList from database. So we want to repeat useEffect every time the beers state is updated -> specify that in dependency array

```
    }, [beers])
```